src/constants/translations/en/change-confirm.json

```diff
@@ -0,0 +1,9 @@
+{
+  "title": "Changes to resource may affect courses and cooperation",
+  "descriptionLesson": "You are about to make changes to the resource '{{resource}}'.",
```

Comment:
I would call it `descriptionResource`. Because it can be both a `lesson` and a `quiz` and an `attachment` - we don't know what resource will come. This is just my humble opinion, which is not that important or critical. It's just that, to me, this name is a bit misleading 🤔